manage.py

```diff
@@ -240,18 +240,27 @@ def add(ctx, collections, note, name, tables_only, field_counts_option, field_li
         name = f"collection_{'_'.join(str(_id) for _id in sorted(collections))}"
 
     schema = f'view_data_{name}'
+
+    db.execute('CREATE SCHEMA IF NOT EXISTS summaries')
+    db.set_search_path(["summaries"])
+    db.execute('''CREATE TABLE IF NOT EXISTS selected_collections
+                  (id SERIAL, schema TEXT NOT NULL, collection_id INTEGER NOT NULL)''')
+    db.execute('CREATE UNIQUE INDEX IF NOT EXISTS selected_collections_id ON selected_collections (id)')
+    db.execute("""CREATE UNIQUE INDEX IF NOT EXISTS selected_collections_schema_collection_id
```

Comment:
I think we only need an index on schema (in which case it is non-unique).

Reply:
The current uniqueness constraint ought to be true, if the schema is added/removed using manage.py. It will cause database errors if a schema is added after being manually deleted. I'm not sure is this is useful (to left you know the database state is inconsistent rather than carrying on), or something we want to avoid.

Reply:
I'm fine with keeping this index. The error should never occur, since users should always be using the CLI - and if the database is inconsistent, it's good for the CLI to error early.